concourse/steps/component_descriptor.mako

```diff
@@ -5,10 +5,19 @@
 <%
 from makoutil import indent_func
 from concourse.steps import step_lib
+from itertools import zip_longest, chain
+
 descriptor_trait = job_variant.trait('component_descriptor')
 main_repo = job_variant.main_repository()
 main_repo_path_env_var = main_repo.logical_name().replace('-', '_').upper() + '_PATH'
 
+policies = descriptor_trait.validation_policies()
+
+policies_args = []
+for policy in policies:
+  policies_args.append("'--values-policies'")
```

Comment:
for the sake of simplicity / readability, i suggest you add single strs of the form `--values-policies=<policy>`. or just extend the args using a mako-for-loop